website/docusaurus.config.js

```diff
@@ -227,7 +226,7 @@ const config = {
                     aria-label="Star this project on GitHub"
                     class="footer__github-stars"
                   >
-                    <img src="https://img.shields.io/github/stars/prettier/prettier?style=social" alt="Star this project on GitHub" />
+                    <img src="https://img.shields.io/github/stars/prettier/prettier?style=social" loading="lazy" alt="Star this project on GitHub" />
```

Comment:
Not related to spacing, just a lighthouse recommendation: 

![image](https://github.com/user-attachments/assets/9b312dc0-fff3-4e01-b687-40021cfd536b)